libs/cart/state/src/facades/cart/cart-facade.interface.ts

```diff
@@ -228,12 +228,51 @@ export interface DaffCartFacadeInterface<
 	orderResultCartId$: Observable<V['cartId']>;
   hasOrderResult$: Observable<boolean>;
 
-	getCartItemDiscountedTotal(itemId: U['item_id']): Observable<number>;
 	getConfiguredCartItemAttributes(itemId: U['item_id']): Observable<DaffConfigurableCartItemAttribute[]>;
 	getCompositeCartItemOptions(itemId: U['item_id']): Observable<DaffCompositeCartItemOption[]>;
 	isCartItemOutOfStock(itemId: U['item_id']): Observable<boolean>;
 	/**
-	 * The state of a cart item.
+   * The state of a cart item.
 	 */
   getCartItemState(itemId: U['item_id']): Observable<DaffCartItemStateEnum>;
+  /**
+   * Selects the specified item's total discount for the entire line.
```

Comment:
What does "for the entire line" mean?

Reply:
For the line item. I should have said 'row' since that's the terminology that we use. Changed